apps/core/utils.py

```diff
@@ -0,0 +1,2 @@
+def apply_properties_to_table():
```

Comment:
We probably don't need this. You can call the existing function `extract_preview_data` in `utils/extraction.py`.

Reply:
But never mind. I'll fix this.